korge/src/commonMain/kotlin/com/soywiz/korge/tests/ViewsForTesting.kt

```diff
@@ -13,7 +13,7 @@ import com.soywiz.korui.input.*
 import kotlinx.coroutines.*
 import kotlinx.coroutines.timeunit.*
 
-open class ViewsForTesting(val frameTime: Int = 10) {
+open class ViewsForTesting(val frameTime: Long = 10) {
```

Comment:
`frameTime` is going to be small. Usually a few milliseconds. But since `TimeSpan` is already inline this method can be changed to `TimeSpan` instead.